### DISCUSSION TITLE
WALC 0.2.2

### DISCUSSION THREAD

**Discussion by daffamumtaz2361** [cstayyab/WALC#187] [General]
Considering that @cstayyab haven't been able to contribute to WALC, he has made a separate Github Organization and added me as maintainer.

Despite being minor release, this version packs a **lot** of new features (I probably should reconsider the versioning schema). But here's to highlight some.

# What's New

## Support for WhatsApp Multi Device 🥳🎉

WALC is now compatible with WhatsApp Multi Device

## New Notification System 💬🎉

The notification now integrate more to the system and WhatsApp. If you're using KDE Plasma 5.18 or above, you can use inline reply. But you can still use the Mark as read action on other DE. Notification will be correctly formatted (bold, italic, and link only) and grouped.

https://user-images.githubusercontent.com/37852137/164596090-cea95a51-49d0-4deb-9c78-5de9c90b6f6f.mp4

However, I did not test the new notification on other DE, it could be broken there. You could check this option to use the old notification.

![image](https://user-images.githubusercontent.com/37852137/164599563-ac4a7670-e3bd-44cf-9b20-933f57254a88.png)

## App Lock 🔒

Secure your chat from prying eyes 👀! It's not the most secure thing in the world, but at least it will keep most people away. Currently open for suggestions, like how to reset the password (yes, there is currently no easy way to reset the password if you forgot it).

## And other new features:

- Always show Dashboard icon even if whatsapp integration is broken
- Notify and disable some features in dashboard when whatsapp integration is broken (will do in dashboard icon and tray in the future)

## Bug fixes:

- Improve online check in WALC offline page
- WALC will correctly open existing window when launching the app again

<hr /><em>This discussion was created from the release <a href='https://github.com/WAClient/WALC/releases/tag/v0.2.2'>WALC 0.2.2</a>.</em>

**Comment by albakhtari** [comment]
Excellent, I'm working on a AUR PKGBUILD at the moment.
